libs/feedback/src/lib/feedback-widget/feedback-widget.component.html

```diff
@@ -14,56 +14,47 @@ <h2 class="modal-title">Feedback</h2>
       <ng-container *ngIf="(messages$ | async) as message">
         <ng-container *ngIf="message.length">
           <div class="feedback-items">
-            <div class="feedback-item" *ngFor="let item of (messages$ | async)">
-              <div class="header">
-                <span class="name">{{ item.name }}</span>
-                <span class="date"> {{ item.timestamp | date }}</span>
+            <ng-container *ngFor="let item of (messages$ | async)">
+              <div class="feedback-item">
+                <feedback-issue-dropdown [message]="item"></feedback-issue-dropdown>
+                <div class="header">
+                  <span class="name">{{ item.name }}</span>
+                  <span class="date"> {{ item.timestamp | date }}</span>
+                </div>
+                <div class="comment">{{ item.comment }}</div>
               </div>
-              <div class="comment">{{ item.comment }}</div>
-            </div>
+            </ng-container>
+
           </div>
         </ng-container>
       </ng-container>
-      <form
-        [formGroup]="formGroup"
-        novalidate
-        (submit)="submit(); modal.close()"
-        class="form-container"
-      >
-        <input
-          formControlName="name"
-          placeholder="Enter your name"
-          type="text"
-          class="mt-0"
-          i18n
-        />
-        <input
-          formControlName="email"
-          type="email"
-          placeholder="Enter your email"
-        />
-        <small class="text-muted" i18n
-          >Email is optional and, will not displayed
-        </small>
+      <form [formGroup]="formGroup"
+            novalidate
+            (submit)="submit(); modal.close()"
+            class="form-container">
+        <input formControlName="name"
+               placeholder="Enter your name"
+               type="text"
+               class="mt-0"
+               i18n/>
+        <input formControlName="email"
+               type="email"
+               placeholder="Enter your email"/>
+        <small class="text-muted" i18n> Email is optional and, will not displayed</small>
         <div>
-          <textarea
-            i18n-placeholder
-            placeholder="Describe your issue or share your ideas"
-            formControlName="comment"
-            (keydown.control.enter)="submit(); modal.close()"
-          >
+          <textarea i18n-placeholder
+                    placeholder="Describe your issue or share your ideas"
+                    formControlName="comment"
```

Comment:
This is optional, but it's much nicer to have controls on the component and do [formControl]="control", because if you misspell, you'll get a compile time error

Reply:
If you misspel here, you'll get an error message in console
Here is an example: 
![image](https://user-images.githubusercontent.com/40883170/54840364-ec73b480-4cdd-11e9-9e19-a0512b1a0278.png)